rollup.config.js

```diff
@@ -69,52 +62,53 @@ if (buildTarget !== buildTargets.DEV) {
   );
 }
 
-const config = {
-  input: "src/core/index.js",
+const config = [];
+
+if (buildTarget === buildTargets.PROD) {
+  config.push({
+    input: "src/baseCode.js",
+    output: [
+      {
+        file: `${destDirectory}baseCode${minifiedExtension}.js`,
+        format: "iife"
+      }
+    ],
+    plugins: minify ? [...plugins, baseCodeTerser] : plugins
+  });
+}
+
+config.push({
+  input: "src/standAlone.js",
```

Comment:
"standalone" is a single word, so "A" should be lowercase. It looks like you got it correct on the file itself, but maybe just forgot to change it here.